server.js

```diff
@@ -33,37 +35,24 @@ app.use(express.static('public'));
  * HTML Endpoints
  */
 
+// routers 
 app.get('/', function homepage(req, res) {
   res.sendFile(__dirname + '/views/index.html');
 });
 
+app.get('/api', controllers.api.index);
 
-/*
- * JSON API Endpoints
- */
+app.get('/api/bjj', controllers.bjj.index);
+app.post('/api/bjj', controllers.bjj.create);
+app.get('/api/bjj', controllers.bjj.show);
+app.delete('/api/bjj/:bjjId', controllers.bjj.destroy);
 
-app.get('/api', function apiIndex(req, res) {
-  // TODO: Document all your api endpoints below as a simple hardcoded JSON object.
-  // It would be seriously overkill to save any of this to your database.
-  // But you should change almost every line of this response.
-  res.json({
-    woopsIForgotToDocumentAllMyEndpoints: true, // CHANGE ME ;)
-    message: "Welcome to my personal api! Here's what you need to know!",
-    documentationUrl: "https://github.com/example-username/express-personal-api/README.md", // CHANGE ME
-    baseUrl: "http://YOUR-APP-NAME.herokuapp.com", // CHANGE ME
-    endpoints: [
-      {method: "GET", path: "/api", description: "Describes all available endpoints"},
-      {method: "GET", path: "/api/profile", description: "Data about me"}, // CHANGE ME
-      {method: "POST", path: "/api/campsites", description: "E.g. Create a new campsite"} // CHANGE ME
-    ]
-  })
-});
 
 /**********
  * SERVER *
  **********/
 
 // listen on the port that Heroku prescribes (process.env.PORT) OR port 3000
-app.listen(process.env.PORT || 3000, function () {
+app.listen(process.env.PORT || 4000, function () {
   console.log('Express server is up and running on http://localhost:3000/');
```

Comment:
if you update the port, please also update the console.log to reflect the correct port!